finality-aleph/src/finalization.rs

```diff
@@ -254,20 +250,24 @@ impl<
             loop {
                 tokio::select! {
                     multisigned_hash = self.rmc.next_multisigned_hash() => {
-                            let unchecked = multisigned_hash.into_unchecked();
+
+                            //let unchecked = multisigned_hash.into_unchecked().clone();
+                            //debug!(target: "afa", "New multisigned_hash {:?}.", unchecked);
```

Comment:
Shouldn't we log this information here?

Reply:
Also, can we not have a commented out code? ;)

Reply:
Fixed.